draft-ietf-core-resource-directory.md

```diff
@@ -1703,6 +1703,51 @@ Care must be taken in such setups to determine the applicable access control mea
 One easy way to do that is to mandate the use of the sector parameter on all operations,
 as no credentials are suitable for operations across sector borders anyway.
 
+## First-Come-First-Remembered: A default policy
+
+The First-Come-First-Remembered policy is provided both as a reference example for a security policy definition,
+and as a policy that implementations may choose to use as default policy in absence of other configuration.
+It is designed to enable efficient discovery operations even in ad-hoc settings.
+
+Under this policy, the RD accepts registrations for any endpoint name that is not assigned to an active registration resource,
+and only accepts registration updates from the same endpoint.
+The policy is minimal in that towards lookup clients it does not make any of the claims of {{entered-resources}} and {{link-confidentiality}},
+and its claims on {{secure-ep}} are limited to the lifetime of that endpoint's registration.
+It does, however, guarantee towards any endpoint that for the duration of its registration, its links will be discoverable on the RD.
+
+When a registration or operation is attempted, the RD MUST determine the client's subject name or public key:
+
+* If the client's credentials indicate a subject name certified by an authority the RD recognizes (which may be the system's trust anchor store), that subject name is stored.
+  With COSE based credentials (like in ACE), the name is taken from the Subject (sub) claim; this step fails if no Subject claim is part of the token.
```

Comment:
COSE ≠ CWT; is this about CWT?
"the token" is what (the CWT)?

Reply:
Yes, CWT (or JWT) is what I meant. This also means it's not "like in ACE" but "as common with ACE". Then, "the token" also makes sense.